Support serialization for uintx/intx in weights_only

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #147500

Summary:
Fixing the issue reported by huggingface

Test Plan:
python test/test_serialization.py -k test_serialization_uintx_intx

Reviewers:

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
Support serialization for uintx/intx in weights_only

Summary:
Fixing the issue reported by huggingface

Test Plan:
python test/test_serialization.py -k test_serialization_uintx_intx

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

mikaylagawarecki reviewed (approved):
LGTM other than lint, thanks for fixing this and adding testing!

jerryzh168 committed:
Update on "Support serialization for uintx/intx in weights_only"


Summary:
Fixing the issue reported by huggingface

Test Plan:
python test/test_serialization.py -k test_serialization_uintx_intx

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
@pytorchbot merge
